core/src/main/java/org/opensearch/sql/ast/tree/RareTopN.java

```diff
@@ -57,4 +57,3 @@ public enum CommandType {
     RARE
   }
 }
-
```

Comment:
Stupid question but this PR mentions adding newlines at the end of files but I've noticed that it looks like this PR removes newlines, am I misunderstanding what you mean by newlines?

Reply:
<img width="1350" alt="image" src="https://github.com/Bit-Quill/opensearch-project-sql/assets/104795536/0b531cdc-44d5-42a6-930f-380b85abbf3b">
It appears weird on GitHub, but if you look at the change in an editor it is correct. In the case of RareTopN.java it is removing a new line as it ends with two new lines.

So, yes, it does remove it if more than 1 new line exists.

Reply:
Ah okay thanks for letting me know, just Github being weird